hack/run-dev-kind.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+readonly SUPPORTED_KIND_VERSION="v0.5.1"
+readonly CURRENT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+readonly REPO_ROOT_DIR=$( cd ${CURRENT_DIR}/../ && pwd )
+
+readonly SC_NAMESPACE="catalog"
+readonly SC_CHART_NAME="catalog"
+readonly HB_NAMESPACE="helm-broker"
+readonly HB_CHART_NAME="helm-broker"
+
+source "${CURRENT_DIR}/ci/lib/utilities.sh" || { echo 'Cannot load CI utilities.'; exit 1; }
+source "${CURRENT_DIR}/ci/lib/deps_ver.sh" || { echo 'Cannot load dependencies versions.'; exit 1; }
+
+print_warning() {
+  echo -e "\033[33m $1 \033[39m"
+}
+
+print_error() {
+  echo -e "\033[31m $1 \033[39m"
+}
+
+print_done() {
+  echo -e "\033[32m $1 \033[39m"
+}
+
+kind::check_kind() {
+  if ! which kind >/dev/null; then
+    print_error "Kind is not installed on your host, install it and try again"
+    exit 1
+  fi
+
+  local version=$(kind version)
+  if [[ "${version}" != "${STABLE_KIND_VERSION}" ]]; then
+    print_warning "Your kind is in ${version}. Your version is not equal than the supported version of kind ${STABLE_KIND_VERSION}"
+  fi
+}
+
+helm::check_helm() {
+  if ! which helm >/dev/null; then
+    print_error "Helm is not installed on your host, install it and try again"
+    exit 1
+  fi
+}
+
+install::tiller() {
+  shout '- Installing Tiller...'
+  kubectl --namespace kube-system create sa tiller
+  kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+  helm init --service-account tiller --upgrade --wait
+}
+
+tiller::assert_tiller_is_up() {
```

Comment:
IMO this can be dropped because we already using the `--wait` flag during the init action

Reply:
I prefer to leave it, if everything will be ok then it is only one iteration and we will be sure tiller is up.